package.json

```diff
@@ -856,17 +857,17 @@
                     "items": {
                       "type": "string"
                     },
-                    "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
+                    "description": "Array of symbol server URLs (example: http​://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```

Comment:
Please run `gulp generateOptionsSchema` to fix the example urls in `package.json`

Reply:
I wish we could fix this.  It gets automatically changed every time you install or update a dependency.

Reply:
FYI the update dependencies script takes care of fixing up package.json.